docs/release-notes/mongoid-8.0.txt

```diff
@@ -305,6 +305,50 @@ Mongoid 7 behavior:
   end
 
 
+``#pluck`` on Embedded Criteria Returns Nil Values
+------------------------------------------------
+
+Mongoid 8 fixes a bug where calling ``#pluck`` on a Mongoid::Criteria
+for embedded documents discarded nil values. This behavior was
+inconsistent with both the ``#pluck`` method in ActiveSupport and
+with how ``#pluck`` works when reading documents from the database.
+
+The following example illustrates the bug:
```

Comment:
The purpose of release notes is not to illustrate bugs but to describe new behavior. Please refer to other release notes and 1) adjust the language to focus on new behavior, and 2) adjust the code examples to give Mongoid 8 behavior first.

Reply:
This is done, please check again.